bentoml/_internal/models/model.py

```diff
@@ -539,6 +535,11 @@ def with_options(self, **kwargs: t.Any) -> ModelInfo:
     def to_dict(self) -> t.Dict[str, t.Any]:
         return bentoml_cattr.unstructure(self)  # type: ignore (incomplete cattr types)
 
+    def parse_options(self, options_class: type[ModelOptions]) -> None:
+        object.__setattr__(
+            self, "options", options_class.with_options(**self.options.data)
```

Comment:
I don't think that `options` have attrributes `data`. This only applies to pydantic model only

Reply:
The `data` attribute came from `UserDict` actually, https://docs.python.org/2/library/userdict.html

Reply:
cool cool